.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
+      language_version: python3.6
+      args: [--config=.black-config.toml]
```

Comment:
Don't you need our other arguments like `-S` for leaving the quotes alone and `-l 100` for setting the line length?

Reply:
I realised that we don't really need the full blown config file, we can just use the args. That said, I still can't get it to not fix the setup.py 🤷‍♂

Reply:
Ah sorry my comment was stupid. This is of course handled in the .toml file. I'm fine with keeping the config file for black. After all we also have config files for the other linters.